### ISSUE TITLE
Focus indication in non-active viewlet icons is waaaay too subtle

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode#2325]
You should be able to tell where focus is in a glimpse in the following screenshot. IMHO if it takes more than 2s it's too subtle:

![image](https://cloud.githubusercontent.com/assets/5047891/12581585/e96d3388-c437-11e5-8769-3bafa60498ed.png)

And here's a gif:
![focus-move-subtle](https://cloud.githubusercontent.com/assets/5047891/12581605/11be58c6-c438-11e5-8ad5-f561a5e3e379.gif)

**Comment by bpasero** [comment]
#2376

**Comment by isidorn** [comment]
Duplicate -> removing bug label
